src/main/java/de/tum/in/www1/artemis/service/connectors/BambooUpdateService.java

```diff
@@ -71,17 +71,17 @@ public String updatePlanRepository(String bambooProject, String bambooPlan, Stri
                 // workaround to pass additional fields
                 bambooClient.doWork(args);
 
-                log.debug("Update plan repository for build plan " + bambooProject + "-" + bambooPlan);
-                org.swift.bamboo.cli.objects.RemoteRepository bambooRemoteRepository = bambooClient.getRepositoryHelper().getRemoteRepository(bambooRepositoryName,
-                        bambooProject + "-" + bambooPlan, false);
+                log.debug("Update plan repository for build plan " + bambooPlan);
+                org.swift.bamboo.cli.objects.RemoteRepository bambooRemoteRepository = bambooClient.getRepositoryHelper().getRemoteRepository(bambooRepositoryName, bambooPlan,
```

Comment:
please update the method signature to make sure we use a consistent naming, in particular key vs. name

Reply:
Renamed to `planKey`